### PR TITLE
New option  to allow multiple OP_RETURN in a tapyrus script

### PR DESCRIPTION
**Pull request by Naviabheeman**
(#141) New command line option -datacarriermultiple is added to allow tapyrus scripts to have multiple OP_RETURNs. its default is false.

### CODE REVIEW

**Review by azuchi — CHANGES_REQUESTED** [review]
`multiple OP_RETURN` means tx has multiple outputs that contains `OP_RETURN`, not multiple `OP_RETURN` in a script. multiple `OP_RETURN` in a script is not necessary.

File: src/script/standard.cpp [hunk 1]
```diff
@@ -190,9 +191,13 @@ bool Solver(const CScript& scriptPubKey, txnouttype& typeRet, std::vector<std::v
...
     // script.
-    if (scriptPubKey.size() >= 1 && scriptPubKey[0] == OP_RETURN && scriptPubKey.IsPushOnly(scriptPubKey.begin()+1)) {
+    if (scriptPubKey.size() >= 1
+     && scriptPubKey[0] == OP_RETURN){
         typeRet = TX_NULL_DATA;
-        return true;
+        if(scriptPubKey.IsPushOnly(scriptPubKey.begin()+1))
+            return true;
+        else
+            return false;
```
Comment (azuchi) [inline src/script/standard.cpp:200]: Why is this change needed?

Comment (Naviabheeman) [reply src/script/standard.cpp:200]: without this change scripts with multiple `OP_RETURN` were **not** marked as `TX_NULL_DATA` and were allowed as standard by the default `TX_CUSTOM` case.

**Review by azuchi — APPROVED** [review]
